healthcheck_wn_condor

```diff
@@ -184,13 +187,16 @@ then
 fi
 # end check if /pool is queryable
 
-# Check if /pool is formatted correctly
-debug "Checking if /pool is formatted correctly"
-xfs_info /pool | grep ftype=1 > /dev/null 2>&1
-RC=$?
-if [ $RC -gt 0 ]
-then
-    fatal_exit "Problem: /pool is not on a properly formatted XFS filesystem"
+# Check if /pool is formatted correctly on physical workernodes
+if [[ $VIRTUAL_WORKER -ne 1 ]]
+    then
+    debug "Checking if /pool is formatted correctly"
+    xfs_info /pool | grep ftype=1 > /dev/null 2>&1
+    RC=$?
+    if [ $RC -gt 0 ]
+    then
+        fatal_exit "Problem: /pool is not on a properly formatted XFS filesystem"
+    fi
```

Comment:
VMs might not have the mountpoint, but the requirements on the underlying filesystem are still valid.
You could find the underlying filesystem with `df --output=target /pool | tail -n 1` and check that it is either `ext4` or `xfs` with `ftype=1`.